src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Security/AuthorizationHandlers/UpdateSpecificReleaseAuthorizationHandlerTests.cs

```diff
@@ -1,268 +1,329 @@
-            GetEnumValues<ReleaseStatus>().ForEach(
-                status =>
-                {
-                    var release = new Release
+            [Fact]
+            public async Task UpdateSpecificReleaseAuthorizationHandler_ReleasePublishingNotStarted()
+            {
+                // Assert that only users with the "UpdateAllReleases" claim can
+                // update an arbitrary Release if it has not started publishing
+                await GetEnumValues<ReleaseStatus>().ForEachAsync(
+                    async status =>
                     {
-                        Id = Guid.NewGuid(),
-                        Status = status
-                    };
+                        var release = new Release
+                        {
+                            Id = Guid.NewGuid(),
+                            Status = status
+                        };
 
-                    var releaseStatusRepository = new Mock<IReleaseStatusRepository>();
+                        await AssertReleaseHandlerSucceedsWithCorrectClaims<UpdateSpecificReleaseRequirement>(
+                            HandlerSupplier(release),
+                            release,
+                            SecurityClaimTypes.UpdateAllReleases
+                        );
+                    }
+                );
+            }
+            
+            [Fact]
+            public async Task UpdateSpecificReleaseAuthorizationHandler_ReleasePublishing()
+            {
+                // Assert that no users can update an arbitrary Release
+                // if it has started publishing
+                await GetEnumValues<ReleaseStatus>().ForEachAsync(
+                    async status =>
+                    {
+                        var release = new Release
+                        {
+                            Id = Guid.NewGuid(),
+                            Status = status,
+                        };
 
-                    releaseStatusRepository.Setup(
-                            s => s.GetAllByOverallStage(
-                                release.Id,
-                                ReleaseStatusOverallStage.Started,
-                                ReleaseStatusOverallStage.Complete
-                            )
-                        )
-                        .ReturnsAsync(new List<PublisherReleaseStatus>());
+                        await AssertReleaseHandlerSucceedsWithCorrectClaims<UpdateSpecificReleaseRequirement>(
+                            HandlerSupplierWhenPublishing(release),
+                            release
+                        );
+                    }
+                );
+            }
+            
+            [Fact]
+            public async Task UpdateSpecificReleaseAuthorizationHandler_ReleasePublished()
+            {
+                // Assert that no users can update an arbitrary
+                // Release if it has been published
+                await GetEnumValues<ReleaseStatus>().ForEachAsync(
```

Comment:
I think you are right, the `Live` property of Release doesn't use the Status field. It's only bothered about whether Published date has been set because that gets set during publishing. Setting the status back to Draft won't unpublish it because unpublishing requires manual intervention.

All the tests in this class iterate over the statuses in the same way because we know the logic of the `UpdateSpecificReleaseAuthorizationHandler` makes use of the ReleaseStatus field and we want to make sure that altering the status to any value has no bearing on the outcome except only when it should do (e.g. when the status is Approved, making sure it passes for the Approver role).